CHANGES.md

```diff
@@ -1,5 +1,7 @@
 ## 0.0.51
 
+-   Changed the migrators to look for "password" instead of "postgres-password" in the cloudsql-db-credentials secret.
```

Comment:
By the way, all other place (line 33: ) is `password ` so probably change here to `password` (rather than from `postgres-password` to `postgres`) could be better

Reply:
Oops! fixed